language/ir-testsuite/tests/signer/runtime_move_to.mvir

```diff
@@ -0,0 +1,62 @@
+module M {
+    resource R1 { f: bool }
+    resource R2<T> { f: T }
+
+    public store(sender: &signer, f: bool) {
+        move_to<R1>(copy(sender), R1 { f: move(f) });
+        return;
+    }
+
+    public store_gen<T>(sender: &signer, t: T) {
+        move_to<R2<T>>(copy(sender), R2<T> { f: move(t) });
+        return;
+    }
+
+    // TODO migrate to use signer
```

Comment:
Ah, does this not work because you need `Signer::address(signer)`?

Reply:
Correct